riker/api.py

```diff
@@ -30,7 +30,7 @@
 from retry import synchronize
 
 import fabric
-fabric.state.output.everything = False
+fabric.state.output.everything = True
```

Comment:
I'm not the biggest fan of OUTPUT-AS-VERBOSE-AS-YOU-POSSIBLY-CAN, but with this set to false, then any time i ran a successful ```riker dokku "<cmd>" --app <app> --env <env>```, I got no output in my terminal's stdout.  fabric was sucking up everything.

Flipping the logging here to True is a dumb hammer.  it let's me use the ```riker dokku``` command, and it is also extremely helpful to see the buildstep output building my npm applications during the deploy process.

This ```fabric.state.output.everything = True``` hammer is a little too verbose, and I am unfortunately not all that familiar with fabric.  Are you aware of any different settings that would be more useful and keep the output pretty?